src/routes/Routes.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { Route, Routes } from "react-router-dom";
+import {Mockbee,MockMan} from "../pages"
+const RoutesList=()=>{
+    return(
+        <Routes>
+            <Route path="/mockbee" element={<Mockbee />} />
```

Comment:
You can use code formatting tools to take care of indentations.

Reply:
Yes this formatting issue has got resolved here with the [#4903a4b](https://github.com/dev-enforced/eleganz-store/pull/2/commits/4903a4b7bca62c44add0d88bfa2e1fc0b219fb1c) and [#a91688](https://github.com/dev-enforced/eleganz-store/pull/2/commits/a9168880851bbe469add64cbcfd8e9c3f29dcf4d)